### DISCUSSION TITLE
circular reference 的翻譯

### DISCUSSION THREAD

**Discussion by husohome** [python/python-docs-zh-tw#530] [Add Term]
### 初步確認

- [X] 我已經確認過 [術語列表](https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8) 中沒有相關術語。
- [X] 我已經確認過 [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html) 中沒有相關術語。

### 術語原文

circular reference

### 術語翻譯

循環參照

### 說明

雖然教育部樂詞網是說「循環參考」
但是好像大家比較會習慣的是Microsoft Excel也會出現的「循環參照」

### 參考資料

https://support.microsoft.com/zh-tw/office/%E7%A7%BB%E9%99%A4%E6%88%96%E5%85%81%E8%A8%B1%E5%BE%AA%E7%92%B0%E5%8F%83%E7%85%A7-8540bd0f-6e97-4483-bcf7-1b49cd50d123
https://terms.naer.edu.tw/search/?csrfmiddlewaretoken=zrrzk23qcFTaTyIPw1DANglfyuddXGl0GdO0eCw7UNXKh3JCmGWAOYUmhtrFApms&match_type=phrase&query_op=&query_field=title&query_term=circular+reference&filter_term=&filter_field=&filter_op=&filter_bool=&filter_term=&filter_field=&filter_op=&filter_bool=&filter_term=&filter_field=&filter_op=&filter_bool=&filter_term=&filter_field=&filter_op=&filter_bool=

**Comment by mattwang44** [comment]
「循環參照」+1
目前 repo 內用到 `reference` 的地方都是翻成 `參照`
